usig/sgx/usig-enclave.go

```diff
@@ -249,3 +335,49 @@ func sgxUint8SliceToBigInt(sgxArr []C.uint8_t) *big.Int {
 
 	return new(big.Int).SetBytes(bytes)
 }
+
+func dlopen() (unsafe.Pointer, error) {
+	f := C.CString(untrustedFile)
+	defer C.free(unsafe.Pointer(f))
+
+	handle := C.dlopen(f, C.RTLD_LAZY)
```

Comment:
Maybe we should use `RTLD_NOW` as in https://golang.org/src/plugin/plugin_dlopen.go.

Reply:
I will update with `RTLD_NOW|RTLD_GLOBAL`, same as `plugin`.

`LAZY` has better performance, but `NOW` is safer in terms of getting an error at `dlopen()` for undefined symbols. If there are many functions which is not referenced, using `NOW` has negative impact on performance. But I think this is not applicable to this project.